ovirtlago/cmd.py

```diff
@@ -240,6 +241,21 @@ def do_ovirt_start(prefix, with_vms, **kwargs):
             with LogTask('Starting Engine VMs'):
                 prefix.virt_env.engine_vm().start_all_vms()
 
+    LOGGER.info(
+        """
+oVirt environment is ready to be used.
+You can access the web UI with the following link and credentials:
+https://{ip}
+Username: {username}
+Password: {password}
+        """.format(
+            ip=prefix.virt_env.engine_vm().ip(),
+            username=constants.ENGINE_USER.split('@')[0],
+            password=prefix.virt_env.engine_vm()
+            .metadata['ovirt-engine-password']
+        )
+    )
+
```

Comment:
Again just a nit, but rather use ```textwrap.dedent``` to not break the function indentation:

i.e.
```python
In [7]: import textwrap

In [8]: def some_text():
   ...:     msg="""
   ...:     hey
   ...:     jude
   ...:     """
   ...:     print textwrap.dedent(msg)
   ...:     

In [9]: some_text()

hey
jude
```